File: android/src/main/kotlin/kent/chien/flutter_tappay_sdk/models/TapPaySdkCommonResult.kt
package kent.chien.flutter_tappay_sdk.models

data class TapPaySdkCommonResult(
  val success: Boolean,
  val message: String?,
) {
  fun toHashMap(): HashMap<String, Any?> {
    val result = HashMap<String, Any?>()
    result["success"] = success
    result["message"] = message ?: ""

    return result
  }
}

File: ios/Classes/models/TapPaySdkCommonResult.swift
//
//  TapPaySdkCommonResult.swift
//  flutter_tappay_sdk
//
//  Created by Kent C on 2023/12/13.
//

class TapPaySdkCommonResult {
  var success: Bool
  var message: String?
  
  init(success: Bool, message: String?) {
    self.success = success
    self.message = message
  }
  
  func toDictionary() -> [String: Any?] {
    return [
      "success": success,
      "message": message
    ]
  }
}
